internal/docs/websocket/WebSocketJavadocAssertions10.xml

```diff
@@ -32,7 +32,7 @@
                             An optional custom configurator class that the developer would like to use  to provide custom configuration of new instances of this endpoint.
                              The implementation  creates a new instance of the configurator per logical endpoint.
                         </description>
-<package>javax.websocket</package>
```

Comment:
The WebSocketJavadocAssertions10.xml  file need not be changed, they actually correspond to the pre-Jakartaee version. It should also be fine to delete the file.

Reply:
Done. Thanks